engines/python/setup/djl_python/deepspeed.py

```diff
@@ -441,9 +442,13 @@ def inference(self, inputs: Input):
                                 batch_index=i)
                     idx += 1
 
-            content_type = self.rolling_batch.get_content_type()
-            if content_type:
-                outputs.add_property("content-type", content_type)
+                formatter = parameters[i].get("output_formatter")
+                content_type = get_content_type_from_output_formatter(
```

Comment:
could we make content-type a property of the request object that we determine based on the output_formatter passed in, and then return it as part of the result from the inference call?

Reply:
Yes it's already returned in the response in the property.